util/cron/test-perf.cray-cs.ofi.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Run performance tests with comm=ofi on a cray-cs system.
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs'
+
+source $CWD/common-perf.bash
+
```

Comment:
Is it unnecessary to source ``common-ofi`` here because we know this machine is easier to use with slurm?

Reply:
Right, it's known that we'll be using a slurm system with the necessary pmix module etc.